.github/workflows/release.yml

```diff
@@ -0,0 +1,388 @@
+
+      - name: Everest - create and update release branch
+        run: |
+          # Check if the branch already exists
+          git fetch
+          check_branch=$(git ls-remote --heads origin ${BRANCH_NAME})
+
+          if [[ -z ${check_branch} ]]; then
+            git checkout -b $BRANCH_NAME
+            git push origin $BRANCH_NAME
+          fi
+          git checkout $BRANCH_NAME
+          
+          # Update deploy manifest 
+          if [[ $env.IS_RC ]]; then 
+            sed -i "s/perconalab\/everest.*/perconalab\/everest:$VERSION/g" deploy/quickstart-k8s.yaml
+            sed -i "s/percona\/percona-everest.*/perconalab\/everest:$VERSION/g" deploy/quickstart-k8s.yaml
+          else
+            sed -i "s/perconalab\/everest.*/percona\/percona-everest:$VERSION/g" deploy/quickstart-k8s.yaml
+            sed -i "s/percona\/percona-everest.*/percona\/percona-everest:$VERSION/g" deploy/quickstart-k8s.yaml
+          fi 
+          
+          # Update the operator go module to reference the version tag
+          go get github.com/percona/everest-operator@$GH_TAG
+          go mod tidy
+          
+          # Change version in Makefile
+          sed -i "s/RELEASE_VERSION ?=.*/RELEASE_VERSION ?= v$VERSION/g" Makefile
+          
+          # if there is something to commit, commit it and add the tag 
+          if [[ -n $(git status --porcelain) ]]; then
+            if git tag --list | grep -q "^$GH_TAG$"; then
+              echo "The tag is already present in github. Please create a different RC/Release"
+              exit 1
+            fi
+            # configure userdata for commits
+            git config --global user.email "everest-ci@percona.com"
+            git config --global user.name "Everest RC CI triggered by ${{ github.actor }}"
+
+            # commit and push the updated files
+            git commit -a -m "update version tag"
+            git push origin $BRANCH_NAME
+          
+            git tag $GH_TAG
+            git push origin $GH_TAG
+          fi
+
+      - name: Everest UI - setup pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 8
+
+      - name: Everest UI - run with Node 20
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: 'pnpm'
+          cache-dependency-path: ui/pnpm-lock.yaml
+
+      - name: Everest UI - build
+        run: |
+          cd ui
+          pnpm install
+          EVEREST_OUT_DIR=${GITHUB_WORKSPACE}/public/dist/ pnpm build
+
+      - name: Everest - build binary
+        run: |
+          CGO_ENABLED=0 GOOS=linux GOARCH=amd64 make build
+
+      - name: Everest - setup docker build metadata
+        uses: docker/metadata-action@v5
+        id: everest_meta
+        with:
+          images: |
+            percona/percona-everest,enable=${{ env.IS_RC == 0 }}
+            perconalab/everest
+          tags: |
+            type=raw,value=${{ env.VERSION }}
+            type=raw,value=latest
```

Comment:
When will we use the latest tag?

Reply:
Every time the workflow runs it will push the latest tag to the `perconalab/everest` repo. In case it's a release run it also pushes latest to `percona/percona-everest`